python/mutable_default_arguments.md

```diff
@@ -0,0 +1,52 @@
+# Mutable default arguments
+
+__NOTE__: Information taken from [python guide](http://docs.python-guide.org/en/latest/writing/gotchas/)
+
+One of the most confusing moments for new developers is when they discover for
+the first time how Python treat default arguments in function definitions. 
```

Comment:
treats, with an S. Now that I re-read it, we can lose "for the first time" since that's what "discover" means:

> One of the most confusing moments for new developers is when they discover how
> Python treats default arguments in function definitions.